docs/source/_static/hackrtd.css

```diff
@@ -110,3 +110,20 @@ a.trio-help-hint, .trio-help-hint a:link, .trio-help-hint a:visited {
     text-decoration: none;
     border-bottom: 1px solid;
 }
+
+/* override table width restrictions */
```

Comment:
Can we move the comment for `th` and place it here to explain both the td and th changes?

Reply:
Something like "Since we have long names compared to other projects, allow word-wrapping in header cells and data cells to avoid too large tables"? (I'm not a native speaker, feel free to rephrase or put something else entirely.)